engine/src/main/resources/templates/Policies/instantEmailTitleV2.txt

```diff
@@ -0,0 +1 @@
+Instant notification - Policies - Red Hat Enterprise Linux
```

Comment:
The old title contained useful information from a customer perspective. Is that okay to replace it with a generic title which doesn't contain that information anymore? Same question from the daily title.

Reply:
According UXD (https://docs.google.com/document/d/17Lba2pepEnbem5LLdLxtqPPp_22Bjo38PQVLUXX5vCQ/edit): 
_EMAIL SUBJECT possibilities:
Type of email - Event name - App - Bundle
8 new recommendations - Advisor
Red Hat Enterprise Linux - Advisor - 8 new recommendations
Instant Notification - New recommendation- Red Hat Enterprise Linux - Advisor (?)
How critical it is -_

Reply:
According UXD: 
_EMAIL SUBJECT possibilities:
Type of email - Event name - App - Bundle
8 new recommendations - Advisor
Red Hat Enterprise Linux - Advisor - 8 new recommendations
Instant Notification - New recommendation- Red Hat Enterprise Linux - Advisor (?)
How critical it is -_